src/firmware/app/logger/logger.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+#include "shared/proto/robot_log_msg.nanopb.h"
+
+/**
+ * Initializes the logger with the given robot id and the log handler function
+ *
+ * @param robot_id The robot id the logs are coming from
+ * @param robot_log_msg_handler The function to handle RobotLog protos
+ */
+void app_logger_init(unsigned robot_id,
+                     void (*robot_log_msg_handler)(TbotsProto_RobotLog log_msg));
+
+/**
+ * Creates a new RobotLog proto and calls the robot_log_msg_handler
+ *
+ * NOTE: If the log msg is longer than the max_size defined in the proto
+ * the log will be truncated.
+ *
+ * @param file_name The file this log is located
+ * @param line_number The line number where this log is located
```

Comment:
Nit:

Kinda felt like the 'line number where the log is located` sounds a bit clunky.

Maybe 'line where the log will be printed?', also what if there is a newline character, is it possible to be on multiple lines.

Reply:
fixed!

Reply:
fixed